src/content/edit-user-script.css

```diff
@@ -63,3 +118,6 @@ body, #editor {
 .CodeMirror {
   height: 100% !important;
 }
+.CodeMirror pre {
+  line-height: 14px !important;
```

Comment:
Why? Does this have anything to do with showing a modal dialog?  Some side effect of now including shoelace?

Reply:
Slide effect of shoelace.